### ISSUE TITLE
Turning off realtime mode leaves electrodes in actuated state (Trac #176)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#178]
When a user clicks on the `realtime mode` checkbox, all switches should be set to their off state.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/176
```json
{
    "status": "closed", 
    "changetime": "2015-04-17T05:34:58", 
    "description": "When a user clicks on the `realtime mode` checkbox, all switches should be set to their off state.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1429248898223719", 
    "component": "dmf_control_board_plugin", 
    "summary": "Turning off realtime mode leaves electrodes in actuated state", 
    "priority": "major", 
    "keywords": "", 
    "version": "1.0", 
    "time": "2015-03-11T07:45:53", 
    "milestone": "", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20150311T07:46:16: ryan changed component from "microdrop" to "dmf_control_board_plugin"

**Comment by cfobel** [comment]
Trac update at 20150417T05:33:48: ryan commented:

> In changeset a2602dca095485542c8d7ebe346d26deb97f0df4:
> ```
> #!CommitTicketReference repository="dmf_control_board_plugin" revision="a2602dca095485542c8d7ebe346d26deb97f0df4"
> Turn off all switches when realtime mode is disabled
> 
> Fix #178
> ```
> 
> 
> 
>

**Comment by cfobel** [comment]
Trac update at 20150417T05:34:58: 
* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"
